cumulus/parachains/integration-tests/chopsticks/tests/v5-tests/index.ts

```diff
@@ -0,0 +1,389 @@
+import { test, expect } from "bun:test";
+import {
+	wnd_ah,
+	XcmV3Junctions,
+	XcmV3Junction,
+	XcmV3MultiassetFungibility,
+	XcmV4Instruction,
+	XcmV3WeightLimit,
+	XcmV3MultiassetMultiAssetFilter,
+	XcmV4AssetAssetFilter
+} from "@polkadot-api/descriptors";
+import { Binary, Enum, createClient } from "polkadot-api";
+import { getWsProvider } from "polkadot-api/ws-provider/web";
+import { withPolkadotSdkCompat } from "polkadot-api/polkadot-sdk-compat";
+import { sr25519CreateDerive } from "@polkadot-labs/hdkd";
+import {
+	DEV_PHRASE,
+	entropyToMiniSecret,
+	mnemonicToEntropy,
+} from "@polkadot-labs/hdkd-helpers";
+import { getPolkadotSigner } from "polkadot-api/signer";
+
+const WESTEND_NETWORK = Uint8Array.from([225, 67, 242, 56, 3, 172, 80, 232, 246, 248, 230, 38, 149, 209, 206, 158, 78, 29, 104, 170, 54, 193, 205, 44, 253, 21, 52, 2, 19, 243, 66, 62]);
+// TODO: find a way to extract keys below from yaml config.
+const BOB_KEY = "5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty";
```

Comment:
I know that from `aliceSigner/bobSigner` you can only get a `Uint8Array` public key; is it possible to use it in place of these addresses?

E.g. `getFreeBalance(AHApi, BOB_KEY);` becomes `await getFreeBalance(AHApi, bobSigned.publicKey);`